cmd/kubelet/app/options/options.go

```diff
@@ -94,6 +95,11 @@ func NewKubeletServer() *KubeletServer {
 	api.Scheme.Default(versioned)
 	config := componentconfig.KubeletConfiguration{}
 	api.Scheme.Convert(versioned, &config, nil)
+
+	config.CAdvisorStatsCacheDuration = metav1.Duration{Duration: 2 * time.Minute}
```

Comment:
Thanks for your feedback. @dixudx 😄 
While I think,  in kubelet/cadvisor_linux.go, we could not judge whether these values are not set.
For example, if a user does not set CAdvisorStatsCacheDuration and another user sets it to be 0, for `cadvisor_linux.go` it can not distinguish two above cases. As a result, I do not think using default constant values in `cadvisor_linux.go` is proper.

In addition, I found that ControllerManager initialize default data for configs in options.go, so I try the same way here.

And feel free to point out if I missed something.